pkg/system/phase2_creating.go

```diff
@@ -93,6 +94,13 @@ func (r *Reconciler) ReconcilePhaseCreatingForMainClusters() error {
 		return err
 	}
 
+	if r.CoreAppConfig.Data["NOOBAA_LOG_LEVEL"] == "warn" {
```

Comment:
This will only be running on noobaa CR reconcile. I think it should work because AFAIR we reconcile noobaa CR on every system store update notification, but maybe we should think of a better solution

Reply:
I think we do enqueue a reconcile if noobaa owned configmap is updated.

Reply:
this time it works - its a fix for my previous PR which wasn't actually running. For sure we can have a better solution - I think we need to completely redesign our logging for both operator and core.